examples/VREP/adaptive_reach.py

```diff
@@ -0,0 +1,120 @@
+"""
+Move the UR5 VREP arm to a target position.
+The simulation ends after 1.5 simulated seconds, and the
+trajectory of the end-effector is plotted in 3D.
+"""
+import numpy as np
+import traceback
+
+# from abr_control.arms import ur5 as arm
+from abr_control.arms import jaco2 as arm
+# from abr_control.arms import onelink as arm
+from abr_control.controllers import OSC, signals
+from abr_control.interfaces import VREP
+
+#backend='nengo'
+backend='nengo_spinnaker'
+# initialize our robot config for the jaco2
+robot_config = arm.Config(use_cython=True, hand_attached=True)
+
+# get Jacobians to each link for calculating perturbation
+J_links = [robot_config._calc_J('link%s' % ii, x=[0, 0, 0])
+           for ii in range(robot_config.N_LINKS)]
+
+# instantiate controller
+ctrlr = OSC(robot_config, kp=200, vmax=0.5)
+
+if backend == 'nengo_spinnaker'
+    # 1 sec of simulation takes 5 minutes 30 seconds, adjust
+    # learning rate since spinnaker runs in real time
+    # spinnaker_run_time/sim_run_time = (1sec/sec)/(330sec/sec)
+    time_scale = 1/330
+else:
+    time_scale = 1
+
+# create our adaptive controller
+adapt = signals.DynamicsAdaptation(
+    robot_config, backend='nengo_spinnaker',
```

Comment:
want `nengo` as the default backend

Reply:
I noticed that in the means_and_scales branch. There are a few example fixups in there